.github/workflows/ci.yaml

```diff
@@ -25,15 +25,15 @@ jobs:
         with:
           node-version: 12
       - name: Install
-        run: yarn install --frozen-lockfile
+        run: yarn install --immutable
       - name: Lint
-        run: npx lerna run lint --stream
+        run: yarn run lint
```

Comment:
1. `lerna` doesn't work with the `workspace:` (see below)
2. yarn v2 has a plugin which allows running a command in all/most workspaces (`yarn workspaces foreach`). These yarn scripts here make use of that.

Reply:
Decided to get rid of `workspace:` protocol, but kept these changes in order to lessen our dependence on `lerna` (only used for `publish`).